core/src/main/scala/org/apache/spark/network/netty/NettyBlockTransferService.scala

```diff
@@ -30,6 +35,7 @@ import org.apache.spark.network.shuffle.{RetryingBlockFetcher, BlockFetchingList
 import org.apache.spark.network.shuffle.protocol.UploadBlock
 import org.apache.spark.serializer.JavaSerializer
 import org.apache.spark.storage.{BlockId, StorageLevel}
+import org.apache.spark.SparkEnv
```

Comment:
nit: import ordering (here and many other places).  I highly recommend the IntelliJ plugin (**not** the built-in ordering) as documented [on the wiki](https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports)